packages/components/src/range-control/README.md

```diff
@@ -1,17 +1,101 @@
+4. **Icon** (optional): An icon can be displayed before or after the range slider.
+5. **Tick mark** (optional): Tick marks represent predetermined values to which the user can move the slider.
+
+### Behavior
+
+- **Click and drag**: The range slider is controlled by clicking the thumb and dragging it.
+- **Click jump**: The range slider is controlled by clicking the track.
+- **Click and arrow**: The continuous slider is controlled by clicking the thumb, then using arrow keys to move it.
+- **Tab and arrow**: The range slider is controlled by using the tab key to select the thumb of the desired slider, then using arrow keys to move it.
+- **Tick marks** (Optional) Discrete sliders can use evenly spaced tick marks along the slider track, and the thumb will snap to them. Each tick mark should change the setting in increments that are discernible to the user.
+- **Value entry field** (Optional): Discrete sliders have value entry fields. After a text entry is made, the slider position automatically updates to reflect the new value.
+
+### Usage
+
+RangeControls reflect a range of values along a track, from which users may select a single value. They are ideal for adjusting settings such as volume, opacity, or text size.
+
+RangeControls can have icons on both ends of the track that reflect a range of values.
+
+#### Immediate effects
+
+Changes made with RangeControls are immediate, allowing a user to make adjustments until finding their preference. They shouldn’t be paired with settings that have delays in providing feedback.
+
+![A range slider that requires a save action](https://make.wordpress.org/design/files/2018/12/rangecontrol-save-action.png)
+
+**Don’t**
+Don’t use RangeControls if the effect isn’t immediate.
+
+#### Current state
+
+Range sliders reflect the current state of the settings they control.
+
+#### Values
+
+![](https://make.wordpress.org/design/files/2018/12/rangecontrol-field.png)
+
+A range slider with an editable numeric value
+
+**Editable numeric values**: Editable numeric values allow users to set the exact value of a range slider. After setting a value, the thumb position is immediately updated to match the new value.
+
+![A range slider with only two values](https://make.wordpress.org/design/files/2018/12/rangecontrol-2-values.png)
+
+**Don’t**
+RangeControls should only be used for choosing selections from a range of values (e.g., don’t use a range slider if there are only 2 values).
+
+![](https://make.wordpress.org/design/files/2018/12/rangecontrol-disabled.png)
+
+**Don’t**
+Sliders should provide the full range of choices available for the user to select from (e.g., don’t disable only part of a range slider).
+
+#### Types
+
+##### Continuous sliders
+
+Continuous sliders allow users to select a value along a subjective range. They do not display the selected numeric value. Use them when displaying/editing the numeric value is not important, like volume.
```

Comment:
Does the range component even allow this?

Reply:
Not explicitly, although you can set a large max number and simulate a continuous slider. I feel like it will be a needed feature in the near future, given that it's an option in a regular [html slider](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range#Setting_the_value's_granularity). Even though the component doesn't support it fully, I think the design guidelines can be somewhat aspirational in this case.